templates/gsl.Makefile.am

```diff
@@ -324,6 +324,10 @@ function define_cpp_lib_variables(variables, library, make, configure)
         define my.is_function = is_function_dependency(my.dependency)
         define my.is_boost = is_boost_dependency(my.dependency)
         define my.is_boost_lib = is_boost_lib_dependency(my.dependency)
+        define my.is_png = is_png_dependency(my.dependency)
+        define my.is_png_lib = is_png_lib_dependency(my.dependency)
+        define my.is_qrencode = is_qrencode_dependency(my.dependency)
+        define my.is_qrencode_lib = is_qrencode_lib_dependency(my.dependency)
```

Comment:
The only reason boost is explicitly encoded here is because it does not employ the package config generalization. Do **libpng** and **qrencode** not use package config?